symfony/plugins/orangehrmAdminPlugin/Api/LocalizationAPI.php

```diff
@@ -0,0 +1,164 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+namespace OrangeHRM\Admin\Api;
+
+use OrangeHRM\Admin\Api\Model\KeyValueModel;
+use OrangeHRM\Core\Api\CommonParams;
+use OrangeHRM\Core\Api\V2\CrudEndpoint;
+use OrangeHRM\Core\Api\V2\Endpoint;
+use OrangeHRM\Core\Api\V2\EndpointResult;
+use OrangeHRM\Core\Api\V2\Exception\NotImplementedException;
+use OrangeHRM\Core\Api\V2\EndpointResourceResult;
+use OrangeHRM\Core\Api\V2\RequestParams;
+use OrangeHRM\Core\Api\V2\Serializer\NormalizeException;
+use OrangeHRM\Core\Api\V2\Validator\ParamRule;
+use OrangeHRM\Core\Api\V2\Validator\ParamRuleCollection;
+use OrangeHRM\Core\Api\V2\Validator\Rule;
+use OrangeHRM\Core\Api\V2\Validator\Rules;
+use OrangeHRM\Core\Exception\CoreServiceException;
+use OrangeHRM\Core\Traits\Service\ConfigServiceTrait;
+use OrangeHRM\Core\Traits\Service\LocalizationServiceTrait;
+
+class LocalizationAPI extends Endpoint implements CrudEndpoint
+{
+    use ConfigServiceTrait;
+    use LocalizationServiceTrait;
+
+    public const PARAMETER_LANGUAGE = 'language';
+    public const PARAMETER_DATE_FORMAT = 'dateFormat';
+    public const PARAMETER_USE_BROWSER_LANGUAGE = 'useBrowserLanguage';
+    public const PARAMETER_BROWSER_LANGUAGE = 'browserLanguage';
+
+    /**
+     * @throws NormalizeException
+     * @throws CoreServiceException
+     */
+    public function getOne(): EndpointResourceResult
```

Comment:
Can you use the below doc comment for all implemented method from `CrudEndpoint`
```php
/**
 * @inheritDoc
 * /
```